src/main/java/org/launchcode/techjobs/persistent/controllers/EmployerController.java

```diff
@@ -1,18 +1,32 @@
 package org.launchcode.techjobs.persistent.controllers;
 
 import org.launchcode.techjobs.persistent.models.Employer;
+import org.launchcode.techjobs.persistent.models.Job;
+import org.launchcode.techjobs.persistent.models.data.EmployerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.validation.Errors;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import java.util.List;
 import java.util.Optional;
 
 @Controller
 @RequestMapping("employers")
 public class EmployerController {
 
+    @Autowired
+    private EmployerRepository employerRepository;
+
+    public String index(Model model) {
+
+        model.addAttribute("employers", "Company Names");
```

Comment:
You'll want to pass in the employers from `employerRepository` here instead of "Company Names" for the template. You'll then need to use Thymeleaf to loop through the employers on the `employers/index.html` template.

https://education.launchcode.org/java-web-development/chapters/thymeleaf-views/template-iteration.html